### DISCUSSION TITLE
how to customize the menu bar of electron app?

### DISCUSSION THREAD

**Discussion by pearlights** [luxuereal/budget_nextron_sqlite#19] [Q&A] [answered]
Let us discuss about that.

**Comment by luxuereal** [comment]
well, I solved that in BrowserWindow API of Electron Main Process.
You can remove menu bar in there.

**Reply by luxuereal** [reply, marked as answer]
try as follow:

const mainWindow = createWindow('main', {
    width: 1366,
    height: 768
  });

  if (isProd) {
    await mainWindow.loadURL('app://./home.html');
  } else {
    const port = process.argv[2];
    await mainWindow.loadURL(`http://localhost:${port}/login`);
    mainWindow.setMenuBarVisibility(false);
  }

**Reply by pearlights** [reply]
okay
